examples-gallery/plot_non_contiguous_parameter_estimation.py

```diff
@@ -0,0 +1,209 @@
+parameters. You can then identify the parameters from all measurement trials
+simultaneously by passing the uncoupled differential equations to opty.
+
+For exaple:
+Four measurements of the location of a simple system consisting of a mass
+connected to a fixed point by a spring and a damper are done. The movement
+is in horizontal direction. The the spring constant and the damping coefficient
+will be identified.
+
+
+**State Variables**
+
+- :math:`x_1`: position of the mass of the first measurement trial [m]
+- :math:`x_2`: position of the mass of the second measurement trial [m]
+- :math:`x_3`: position of the mass of the third measurement trial [m]
+- :math:`x_4`: position of the mass of the fourth measurement trial [m]
+- :math:`u_1`: speed of the mass of the first measurement trial [m/s]
+- :math:`u_2`: speed of the mass of the second measurement trial [m/s]
+- :math:`u_3`: speed of the mass of the third measurement trial [m/s]
+- :math:`u_4`: speed of the mass of the fourth measurement trial [m/s]
+
+**Parameters**
+
+- :math:`m`: mass for both systems system [kg]
+- :math:`c`: damping coefficient for both systems [Ns/m]
+- :math:`k`: spring constant for both systems [N/m]
+- :math:`l_0`: natural length of the spring [m]
+
+"""
+# %%
+# Set up the equations of motion and integrate them to get the measurements.
+#
+import sympy as sm
+import numpy as np
+import sympy.physics.mechanics as me
+import matplotlib.pyplot as plt
+from scipy.integrate import solve_ivp
+from opty import Problem
+from opty.utils import parse_free
+
+x1, x2, x3, x4, u1, u2, u3, u4 = me.dynamicsymbols('x1, x2, x3, x4, u1, u2, u3, u4')
+m, c, k, l0 = sm.symbols('m, c, k, l0')
+t = me.dynamicsymbols._t
+
+eom = sm.Matrix([
+    x1.diff(t) - u1,
+    x2.diff(t) - u2,
+    x3.diff(t) - u3,
+    x4.diff(t) - u4,
+    m*u1.diff(t) + c*u1 + k*(x1 - l0),
+    m*u2.diff(t) + c*u2 + k*(x2 - l0),
+    m*u3.diff(t) + c*u3 + k*(x3 - l0),
+    m*u4.diff(t) + c*u4 + k*(x4 - l0),
+])
+# %%
+# Equations of motion.
+sm.pprint(eom)
+
+# %%
+# Create the measurements for this example.
+#
+rhs = np.array([
+    u1,
+    u2,
+    u3,
+    u4,
+    1/m * (-c*u1 - k*(x1 - l0)),
+    1/m * (-c*u2 - k*(x2 - l0)),
+    1/m * (-c*u3 - k*(x3 - l0)),
+    1/m * (-c*u4 - k*(x4 - l0)),
+])
+
+qL = [x1, x2, x3, x4, u1, u2, u3, u4]
+pL = [m, c, k, l0]
+
+rhs_lam = sm.lambdify(qL + pL, rhs)
+
+def gradient(t, x, args):
+    return rhs_lam(*x, *args).reshape(8)
+
+t0, tf = 0, 20
+num_nodes = 500
+times = np.linspace(t0, tf, num_nodes)
+t_span = (t0, tf)
+
+x0 = np.array([2, 3, 4, 5, 0, 0, 0, 0])
+pL_vals = [1.0, 0.25, 1.0, 1.0]
+
+resultat1 = solve_ivp(gradient, t_span, x0, t_eval = times, args=(pL_vals,))
+resultat = resultat1.y.T
+
+measurements = []
+np.random.seed(123)
+for i in range(4):
+    measurements.append(resultat[:, i] + np.random.randn(resultat.shape[0]) * 0.5 +
+        np.random.randn(1)*2)
+
+# %%
+# Set up the Identification Problem.
+# --------------------------------
```

Comment:
The error is here and it is exactly what it says in the error message.

Reply:
> The error is here and it is exactly what it says in the error message.

Fair enough! I only looked at the 'big' title. I will correct and push